apps/studio/src/server/modules/auth/email/__tests__/email.router.test.ts

```diff
@@ -123,6 +125,13 @@ describe("auth.email", () => {
       await expect(result).resolves.toMatchObject(expectedUser)
       // Session should have been set with logged in user.
       expect(session.userId).toEqual(expectedUser.id)
+      // Audit log should have been created.
+      const auditLogs = await db
+        .selectFrom("AuditLog")
+        .selectAll()
+        .executeTakeFirst()
+      expect(auditLogs).toBeDefined()
+      expect(auditLogs?.eventType).toBe(AuditLogEvent.Login)
```

Comment:
thoughts on
- checking that only 1 record is created? ref: https://github.com/opengovsg/isomer/pull/1175#discussion_r2004845813
- also that the JSON logged is correct as much as possible. ref: https://github.com/opengovsg/isomer/pull/1175#discussion_r2004859976

?

Reply:
Let me add for 1.

For 2, I think the JSON logged should be covered under the audit logs service unit test. This part is just the caller for that function hence I only opted to test the event type, but the shape is not this service's responsibility.

Reply:
Done for 1 in 6df2f491137cb7321046a905680739bb66f3f381.